bake/bake.go

```diff
@@ -1338,7 +1338,23 @@ func toBuildOpt(t *Target, inp *Input) (*build.Options, error) {
 	}
 	bo.Platforms = platforms
 
-	bo.SecretSpecs = t.Secrets.ToPB()
+	secrets := t.Secrets
+	if build.IsRemoteURL(bi.ContextPath) {
```

Comment:
I don't think this is correct. I think if you have a subpath in bake and command invoked directly then the contextpath in here is a subdir but the contextstate has been replaced with something like `llb.Git().Copy("subdir")`

Reply:
Ah you right it would not work for this case